README.md

```diff
@@ -1,17 +1,40 @@
-Chitter Challenge
+# Chitter
 =================
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
 
-Challenge:
--------
+## Setting up your databases:
+
+**Create Peep Manager DB**
+
+1. Connect to ```psql``` in your terminal.
+2. Create the database using the psql command ```CREATE DATABASE peeps_manager;```
+3. Connect to the database using the pqsl command ```\c peeps_manager;```
+4. Run the query we have saved in the file ```01_create_peeps_table.sql```
+
+**Create Peeps Test Manager DB**
+
+Do the above again, with the following changes:
+
+- Step 2: ```CREATE DATABASE peeps_test_manager;```
+- Step 3: ``` \c peeps_test_manager;```
+
+## How to Run:
+
+After setting up your databases, run the app using rackup with ```rackup -p 2345```.
+From here, navigate to ```localhost:2345``` in your browser.
+Once you're there, you will be able to see the public Chitter stream (which will be empty, unless you have already added Peeps!), and a 'log in' feature. Type in your username, and click 'log in' to sign in. Now you can add peeps! Don't forget to log out!
 
-As usual please start by forking this repo.
+![chitter homepage](diagrams/chitter-welcome.png)
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+------------
+
+## The Process
+
+I first looked through the features/user stories to create a domain model and diagrams to help structure my database and system. Unfortunately I was only able to successfully implement the first 4 user stories, as I ran out of time and was not able to complete the 'Harder' user stories (signing in and signing out securely). Fortunately, I was able to partially implement these features, and have diagrams in place if I decide to tackle this at a later date.
```

Comment:
Great idea adding a paragraph to summarise your plan and what worked and what needs more research.